### ISSUE TITLE
[BUG][CML-H]“error：can't find BE for DAI ALH0x202.OUT” display after booting up successfully.

### ISSUE THREAD

**Issue by sinahuang** [thesofproject/sof#2569]
**Describe the bug**
“error：can't find BE for DAI ALH0x202.OUT” display after booting up successfully. Aplay via any pipeline after issued occurred, " error: no config for DAI (null)" shows.

**To Reproduce**
1. Boot up system.
2. Check the dmesg.

**Reproduction Rate**
5/5

**Expected behavior**
Device can boot up successfully, no error.

```
[    3.998447] sof-audio-pci 0000:00:1f.3: tplg: ready widget id 21 pipe 4 type 27 name : ALH0x202.OUT stream SDW1-Playback
[    3.998448] sof-audio-pci 0000:00:1f.3: dai ALH0x202.OUT: type 4 index 514
[    3.998449] sof-audio-pci 0000:00:1f.3:  config: periods snk 0 src 2 fmt 1
[    3.998451] sof-audio-pci 0000:00:1f.3: ipc tx: 0x30010000: GLB_TPLG_MSG: COMP_NEW
[    3.998674] sof-audio-pci 0000:00:1f.3: ipc tx succeeded: 0x30010000: GLB_TPLG_MSG: COMP_NEW
[    3.998675] sof-audio-pci 0000:00:1f.3: error: can't find BE for DAI ALH0x202.OUT
[    3.998676] sof-audio-pci 0000:00:1f.3: tplg: ready widget id 22 pipe 4 type 32 name : PIPELINE.4.ALH0x102.OUT stream ALH0x102.OUT
[    3.998677] sof-audio-pci 0000:00:1f.3: tplg: pipeline id 4 comp 22 scheduling comp id 18
[    3.998678] sof-audio-pci 0000:00:1f.3: pipeline PIPELINE.4.ALH0x102.OUT: period 1000 pri 0 mips 5000 core 0 frames 0
[    3.998680] sof-audio-pci 0000:00:1f.3: ipc tx: 0x30100000: GLB_TPLG_MSG: PIPE_NEW
[    3.998903] sof-audio-pci 0000:00:1f.3: ipc tx succeeded: 0x30100000: GLB_TPLG_MSG: PIPE_NEW
[    3.998905] sof-audio-pci 0000:00:1f.3: DSP core(s) enabled? 1 : core_mask 1
```
```
[   74.375839] sof-audio-pci 0000:00:1f.3: ipc tx: 0x80010000: GLB_DAI_MSG: CONFIG
[   74.376101] sof-audio-pci 0000:00:1f.3: ipc tx succeeded: 0x80010000: GLB_DAI_MSG: CONFIG
[   74.376115] sof-audio-pci 0000:00:1f.3: ipc tx: 0x80010000: GLB_DAI_MSG: CONFIG
[   74.376320] sof-audio-pci 0000:00:1f.3: ipc tx succeeded: 0x80010000: GLB_DAI_MSG: CONFIG
[   74.376331] sof-audio-pci 0000:00:1f.3: ipc tx: 0x80010000: GLB_DAI_MSG: CONFIG
[   74.376540] sof-audio-pci 0000:00:1f.3: ipc tx succeeded: 0x80010000: GLB_DAI_MSG: CONFIG
[   74.376543] sof-audio-pci 0000:00:1f.3: error: no config for DAI (null)
[   74.376557] sof-audio-pci 0000:00:1f.3: ipc tx: 0x80010000: GLB_DAI_MSG: CONFIG
[   74.376818] sof-audio-pci 0000:00:1f.3: ipc tx succeeded: 0x80010000: GLB_DAI_MSG: CONFIG
[   74.376833] sof-audio-pci 0000:00:1f.3: ipc tx: 0x80010000: GLB_DAI_MSG: CONFIG
[   74.377085] sof-audio-pci 0000:00:1f.3: ipc tx succeeded: 0x80010000: GLB_DAI_MSG: CONFIG
[   74.377099] sof-audio-pci 0000:00:1f.3: ipc tx: 0x80010000: GLB_DAI_MSG: CONFIG
```

**Impact**
“error：can't find BE for DAI ALH0x202.OUT” display after booting up successfully.

**Test recipe**

Kernel:https://github.com/thesofproject/linux/commits/topic/sof-dev  commit:45f6424
FW: https://github.com/thesofproject/sof/commits/master commit:  a133d5
Tplg: Same with FW branch,  sof-cml-rt711-rt1308-rt715.tplg
Platform:CML-H Laptop with codec ALC711, 2 x RT1308, RT715 in SDW mode.

[dmesg18.log](https://github.com/thesofproject/sof/files/4347072/dmesg18.log)
" error: no config for DAI (null)" happened dmesg:
[dmesg199.log](https://github.com/thesofproject/sof/files/4353133/dmesg199.log)

**Comment by sinahuang** [comment]
Reverted commit :https://github.com/thesofproject/sof/commit/5e8de9fd52e67d96a57445b279b2b7947505bf6e, issue can't reproduce on CML-H.

**Comment by plbossart** [comment]
Yeah, we need to be more careful with topology changes, we need the matching machine driver to be ready as well, or refer to a PR.

**Comment by bardliao** [comment]
It should be "fixed" if we remove the [SOF_SDW_NO_AGGREGATED ](https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c#L247)flag. But we still need #2539 #2540 or alternative PR to make synchronized playback work.

**Comment by tlauda** [comment]
@bardliao Are you going to change something in topology or should we move this bug to linux repo?

**Comment by bardliao** [comment]
> @bardliao Are you going to change something in topology or should we move this bug to linux repo?

Can we ignore this issue and #2570 until #2539 and #2540 are merged? I will send a kernel PR to remove SOF_SDW_NO_AGGREGATED after #2539 and #2540 are merged.

**Comment by bardliao** [comment]
This is due to mismatch between machine driver and topology. It will be fixed by thesofproject/linux#1928

**Comment by sinahuang** [comment]
Issue can't be reproduced with sof-dev (commit:9b5b6c2a3)+pr1928 and master(commit:d12ad78) +pr2569+pr2540+pr2541.
